backends/qualcomm/builders/op_cast.py

```diff
@@ -1,57 +0,0 @@
-# Copyright (c) Qualcomm Innovation Center, Inc.
-# All rights reserved
-#
-# This source code is licensed under the BSD-style license found in the
-# LICENSE file in the root directory of this source tree.
-from typing import Dict
-
-import executorch.backends.qualcomm.python.PyQnnWrapperAdaptor as PyQnnWrapper
-
-import torch
-
-from .node_visitor import NodeVisitor, register_node_visitor
-from .qnn_constants import OpCast, QNN_OP_PACKAGE_NAME_QTI_AISW
-
-
-@register_node_visitor
-class Cast(NodeVisitor):
```

Comment:
What was the reason to remove it?

Reply:
Pardon for missleading. We change this file to [op_to.py](https://github.com/pytorch/executorch/pull/4142/files#diff-1b345d4dcc7c7035fa648723d76b7282e1c54ef06216de42084af1910a912e2c).

The reason is that, we should convert torch to_copy op to two different backend ops, cast or convert conditionally. Therefore we make a new file for it.